react/features/jwt/TokenData.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+import jwtDecode from 'jwt-decode';
+
+/**
+ * Represent the data parsed from the JWT token
+ */
+export default class TokenData {
```

Comment:
The class `TokenData` is obsolete now. Its fields should be the properties of the redux state `features/jwt`.